default.xml

```diff
@@ -4,6 +4,10 @@
            fetch="git://android.git.kernel.org/"
```

Comment:
All PRs should be submitted as Gerrit Changes. Please see the documentation at https://wiki.cyanogenmod.org/w/Doc:_using_gerrit